rust/agents/validator/src/validator.rs

```diff
@@ -105,6 +105,21 @@ impl BaseAgent for Validator {
     async fn run(&self) -> Instrumented<JoinHandle<Result<()>>> {
         self.announce().await.expect("Failed to announce validator");
 
+        let reorg_period = NonZeroU64::new(self.reorg_period);
```

Comment:
nit: We call this "lag" in run_checkpoint_submitters, let's be consistent (I prefer reorg_period)